src/templates/finops-hub/modules/keyVault.bicep

```diff
@@ -86,6 +86,7 @@ resource storageRef 'Microsoft.Storage/storageAccounts@2022-09-01' existing = {
 resource keyVault_secrets 'Microsoft.KeyVault/vaults/secrets@2022-11-01' = {
   name: storageRef.name
   parent: keyVault
+  tags: tags
```

Comment:
Do secrets need tags? We should be good with only the vault instance, right?

Reply:
I just put them on anything that supported tags. I agree we probably don't need them on secrets